CMAKE.md

```diff
@@ -0,0 +1,31 @@
+# Building with CMake
+
+Some portions of this repository are portable beyond iOS and can be built using
+CMake.
+
+## Dependencies
+
+You need:
+  * `cmake`
+  * `ninja` (optional)
```

Comment:
I've removed the `ninja` references in here. It's causing more confusion than it's worth.